### DISCUSSION TITLE
what's the best way to disable alpha-nvim.  I'd like to start in telescope

### DISCUSSION THREAD

**Discussion by jaredmcqueen** [LazyVim/LazyVim#1002] [General]
100% of the time when I open neovim I want to use telescope to look for files to open.  How can I disable alpha-nvim and instead just have telescope open, searching for files within CWD?

**Comment by jaredmcqueen** [comment]
so when I put this in `options.lua` it opens so it works.  But it also opens when i directly edit a file -- like `neovim blah.rs` 
```lua
vim.api.nvim_create_autocmd("VimEnter", {
  callback = function()
    vim.cmd("silent! lua require('telescope.builtin').find_files()")
  end,
})

```
How can I suppress the popup when the buffer contains a real file?

**Comment by jaredmcqueen** [comment]
ok figured it out.  in my `init.lua` file i put:

```lua
if vim.fn.argv(0) == "" then
  require("telescope.builtin").find_files()
end
```

**Comment by jaredmcqueen** [comment]
this was my final solution:
```lua
vim.api.nvim_create_autocmd("VimEnter", {
  callback = function()
    if vim.fn.argv(0) == "" then
      require("telescope.builtin").find_files()
    end
  end,
})
```

**Reply by folke** [reply]
Glad you figured it out! :)

**Reply by RunnersNum40** [reply]
Did you put this in `init.lua` or `autocmds.lua`? I haven't had any luck with either.
